src/components/userauth/ResetPasword/index.css

```diff
@@ -0,0 +1,3 @@
+.loginContainer__alert {
```

Comment:
there is a typo in ResetPasword in the directory

Reply:
PTAL